lib/cmake/legacy/legacymake.cmake

```diff
@@ -0,0 +1,48 @@
+# This file contains legacy cmake scripts that is going to be removed
+# in a future release.
+
+# Add CMake modules.
+list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/legacy/Modules)
+
+# Add compiler flags.
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -O2 -fPIC -Wno-narrowing")
+
+# Include Caffe2 CMake utils.
+include(cmake/legacy/Utils.cmake)
+
+# Find dependencies.
+include(cmake/legacy/Dependencies.cmake)
+
+# Print configuration summary.
+include(cmake/Summary.cmake)
+detectron_print_config_summary()
+
+# Collect custom ops sources.
+file(GLOB CUSTOM_OPS_CPU_SRCS ${CMAKE_CURRENT_SOURCE_DIR}/ops/*.cc)
+file(GLOB CUSTOM_OPS_GPU_SRCS ${CMAKE_CURRENT_SOURCE_DIR}/ops/*.cu)
+
+# Install custom CPU ops lib.
+add_library(
+     caffe2_detectron_custom_ops SHARED
+     ${CUSTOM_OPS_CPU_SRCS})
+
+target_include_directories(
+    caffe2_detectron_custom_ops PRIVATE
+    ${CAFFE2_INCLUDE_DIRS})
+target_link_libraries(caffe2_detectron_custom_ops caffe2)
+install(TARGETS caffe2_detectron_custom_ops DESTINATION lib)
+
+# Install custom GPU ops lib.
+if (HAVE_CUDA)
```

Comment:
Should this be ${HAVE_CUDA} ?

Reply:
Went a bit sloppy, I think both would work...